client/src/components/EventPage/index.jsx

```diff
@@ -52,6 +52,7 @@ class EventPage extends Component {
         this.setState({
           category: data.category,
           tags: data.tags,
+          status_str: data.status_str,
```

Comment:
Same, do not shrink the names.

Reply:
Are you think the str is not a common abbreviation?

Reply:
Same as first comment we can skip this for now.